src/libmongoc/doc/installing.rst

```diff
@@ -127,7 +172,7 @@ If ``cmake`` completed successfully, you will see a considerable amount of outpu
 
 .. parsed-literal::
 
-  -- Build files have been written to: /home/user/mongo-c-driver-x.y.z/cmake-build
+  -- Build files have been written to: /home/user/mongo-c-driver-|version|/cmake-build
 
 If ``cmake`` concludes with anything different, then there is likely an error or some other problem with the build. Review the output to identify and correct the problem.
 
```

Comment:
Below too, "The build will detect" => "The configuration will detect"?

Reply:
Done